pycypher/local_tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Copyright 2017, Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
Should this file perhaps be a Makefile / Makefile.am?

Reply:
This is only for testing, this will not generate build artifacts suitable for uploading. Also, the preferred way is to use linux_build.sh which runs tests on various versions of Python and on 32 and 64 bit arch, it also generates packages ready for upload.

I would leave these two scripts as-is and call one of them from a Makefile so that this can be switched easily.

Reply:
Ok, but having it in a Makefile means the include paths, etc, are already set up. Is "linux_build.sh" and "local_test.sh" a standard in Python packaging? (Apologies for my lack of familiarity). If not, perhaps they can all go into a Makefile? I dislike having arbitrary build scripts lying around when make is really designed for this purpose.

Reply:
No, these are custom scripts. I can put them as commands in a Makefile to be consistent with the rest of the project.

Reply:
I moved the `local_tests.sh` to `tests/` and added it to `TESTS` and to `dist_check_SCRIPTS` in `Makefile.am`. I moved other build scripts to `<repository-root>/ci` folder.